services/blockstorage/adapter/leveldb_persistence.go

```diff
@@ -12,28 +30,138 @@ type levelDbBlockPersistence struct {
 	blockWritten chan bool
 	blockPairs   []*protocol.BlockPairContainer
 	config       Config
+	reporting    instrumentation.BasicLogger
+	db           *leveldb.DB
+}
+
+type config struct {
+	name string
+}
+
+func (c *config) NodeId() string {
+	return c.name
+}
+
+func NewLevelDbBlockPersistenceConfig(name string) Config {
```

Comment:
Please modify this to the standard config mechanism, we don't want to define configs this way per service or adapter. We have a global system config that works through interfaces..